### ISSUE TITLE
VoiceOver cannot announce vertical Scrollbar

### ISSUE THREAD

**Issue by yama3nomori** [flutter/flutter#79355]
TalkBack can announce 'A to B of C' on ListView() when I scrolled lists.
But VoiceOver cannot announce it and focus vertical a scrollbar.
Visual impaired users cannot understand how long they scroll lists.

```
Scrollbar(
child: ListView(
  children: <Widget>[
  Text('item 1'),
  Text('item 2'),
  Text('item 3'),
  // and more items ...
  ],
),
),
```

**Comment by darshankawar** [comment]
@yama3nomori 
Same as https://github.com/flutter/flutter/issues/79354#issuecomment-810086245. Please provide details as requested in your earlier issue.

**Comment by yama3nomori** [comment]
If ListView() is displayed vertical scrollbar, I think this issue don't need.
Unfortunately, I could not see it_I'm blind person.

Does it need Scrollbar() when I use ListView()?

**Comment by darshankawar** [comment]
@yama3nomori 
ListView doesn't explicitly provide a scrollbar and hence you may use `Scrollbar` widget to wrap around `ListView()`.

**Comment by yama3nomori** [comment]
I understand about the scrollbar.

Usually VoiceOver can focus and swipe up or down of one finger gesture to scrollbar.
But Flutter app cannot do that.

**Comment by darshankawar** [comment]
@yama3nomori 
Using your code sample above and with voiceover, I was able to swipe up or down using one finger properly.
Can you elaborate on the issue ?

**Comment by yama3nomori** [comment]
You were able to do that, weren't you?
Maybe it's impossible to do that.
Because VoiceOver cannot focus vertical scrollbar.
If you swipe up or down, you cannot scroll lists.

**Comment by darshankawar** [comment]
Can you check [this](https://github.com/flutter/flutter/issues/79457) issue and see if it is similar to your ?

**Comment by yama3nomori** [comment]
Above issue is different.
ListView() can move items swiping right or left on VoiceOver.
This gesture can focus each item on lists.
 Of course it can scroll lists.

If VoiceOver can focus vertical scrollbar, you can scroll lists with some items to swipe up or down.

**Comment by darshankawar** [comment]
<details>
<summary>flutter doctor -v</summary>

```
[✓] Flutter (Channel master, 2.1.0-13.0.pre.496, on Mac OS X 10.15.4 19E2269
    darwin-x64, locale en-GB)
    • Flutter version 2.1.0-13.0.pre.496 at
      /Users/dhs/documents/fluttersdk/flutter
    • Framework revision 026ee9756c (35 minutes ago), 2021-04-06 21:52:35 -0700
    • Engine revision 92f4dfb048
    • Dart version 2.13.0 (build 2.13.0-208.0.dev)

[!] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.3, Build version 12C33
    ! CocoaPods 1.9.3 out of date (1.10.0 is recommended).
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin
        code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To upgrade see
      https://guides.cocoapods.org/using/getting-started.html#installation for
      instructions.

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] VS Code (version 1.55.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.20.0

[✓] Connected device (3 available)
    • Darshan's iphone (mobile) • 21150b119064aecc249dfcfe05e259197461ce23 • ios
      • iOS 14.4.1
    • macOS (desktop)           • macos                                    •
      darwin-x64     • Mac OS X 10.15.4 19E2269 darwin-x64
    • Chrome (web)              • chrome                                   •
      web-javascript • Google Chrome 89.0.4389.114

! Doctor found issues in 1 category.

```

</details>
